platforms/software/dependency-management/build.gradle.kts

```diff
@@ -91,6 +90,7 @@ dependencies {
     implementation(libs.fastutil)
     implementation(libs.gson)
     implementation(libs.httpcore)
+    implementation("org.apache.commons:commons-text:1.14.0")
```

Comment:
this might be not ideal, but needed to fix deprecation. 

How to apply this properly, please?